sympy/integrals/integrals.py

```diff
@@ -1169,20 +1180,19 @@ def as_sum(self, n=None, method="midpoint", evaluate=True):
         """
         Approximates a definite integral by a sum.
 
-        Arguments
-        ---------
-        n
+        Parameters
+        ==========
+
+        n :
             The number of subintervals to use, optional.
-        method
+        method :
             One of: 'left', 'right', 'midpoint', 'trapezoid'.
-        evaluate
+        evaluate : bool
             If False, returns an unevaluated Sum expression. The default
             is True, evaluate the sum.
 
         These methods of approximate integration are described in [1].
```

Comment:
This needs to be isolated with section

Reply:
I added a notes section for the last line.